packages/ontario-design-system-component-library/src/utils/helper/utils.ts

```diff
@@ -67,7 +67,7 @@ export function removeObjectsBySpecificKey<T>(objects: T[], key: keyof T, value:
  * @returns {boolean}
  */
 export function isNumber(value: string | number): boolean {
-	return isNaN(Number(value)) === true;
+	return isNaN(Number(value)) === false;
```

Comment:
For consistency and clarity, this could be named `isValidNumber`?

Reply:
The goal here is that the `value` is a `number` the validity of the number is up to the caller to figure out.  If it is a number, is it a valid number? This only does the first part of the check.